### ISSUE TITLE
Local Cluster Management - Etcd Data - unknown flag: --host-data-dir

### ISSUE THREAD

**Issue by diegolovison** [openshift/origin#20997]
unknown flag: --host-data-dir

##### Version
oc v3.10.0+dd10d17
kubernetes v1.10.0+b81c8f8
features: Basic-Auth

##### Steps To Reproduce
Following the documentation https://github.com/openshift/origin/blob/master/docs/cluster_up_down.md#etcd-data when running the command we got an error
`oc cluster up --skip-registry-check=true --host-data-dir=host-data-dir/jenkins-slave`

##### Current Result
Error: unknown flag: --host-data-dir

##### Expected Result
We should fix the documentation

**Comment by mfojtik** [comment]
yes, sorry for that... we switched to use the `--base-dir` instead ?

**Comment by jwforres** [comment]
@openshift/sig-master

**Comment by diegolovison** [comment]
Yes. I tested with `--base-dir` and it is working
I decided to create the PR https://github.com/openshift/origin/pull/21001

**Comment by praveenkumar** [comment]
@diegolovison since this is fixed can you close this one? not sure why CI didn't do it.
